observatory-platform/observatory/platform/docker/docker-compose.observatory.yml.jinja2

```diff
@@ -285,7 +286,7 @@ services:
     networks:
       - {{ docker_network_name }}
     healthcheck:
-      test: ["CMD", "pg_isready", "-U", "airflow"]
+      test: ["CMD", "pg_isready", "-U", "{{ postgres_user }}"]
```

Comment:
I never really got to the bottom of my problems, I think they were largely around docker not having enough memory (ES is a memory hog FYI). I actually changed it back to just being airflow and nothing broke. So not sure exactly what is going on

Reply:
It's worth considering making ES+Kibana being an optional flag in the observatory start command (maybe that is already the case, I should double check). Elasticsearch want's to grow to around a 4gb memory footprint, and if you are testing components that don't rely on it, it would be helpful to not turn that on.

Reply:
Yeah we can do that in the future. Could you add a ticket for it on JIRA?

Reply:
yeah, will do